### PR TITLE
list instance creation handled during SetRequest

### PR DESCRIPTION
**Pull request by martin-volf**
The Update path is converted to a JSON object on top of the payload and uploaded via the API.

### CODE REVIEW

**Review by micnovak — APPROVED** [review]

File: src/confd_gnmi_api_adapter.py [hunk 1]
```diff
@@ -745,3 +719,35 @@ def delete(self, prefix, paths):
 
         log.info("==> ops=%s", ops)
         return ops
+
+
+class Updater:
```
Comment (jomiklos) [inline src/confd_gnmi_api_adapter.py:724]: some basic one-line py-doc comments would be nice...
(at least for constructor)

File: src/confd_gnmi_api_adapter.py [hunk 2]
```diff
@@ -745,3 +719,35 @@ def delete(self, prefix, paths):
...
+    def __init__(self, adapter, t, prefix):
```
Comment (jomiklos) [inline src/confd_gnmi_api_adapter.py:725]: "t" is bit cryptic... is this "maapi_trans"?

Comment (martin-volf) [reply src/confd_gnmi_api_adapter.py]: using `trans` elsewhere, so changed to that

File: src/confd_gnmi_api_adapter.py (hunk 1 above)
Comment (martin-volf) [reply src/confd_gnmi_api_adapter.py:724]: Added a docstring to the class.

**Pull request merged**